
import { Temperature } from "@/components/Temperature";
import { Content, Header, AccentLine, Dropdown, FootnotesProvider, Footnote, FootnotesList, FootnoteDefinition  } from "@/components";

<Header>

# dry lab model
MS2 VLP stability analysis
</Header>

<FootnotesProvider>
<Content items="left">
<AccentLine/>
# MD simulations to investigate the stability of MS2 VLPs




The capacity for **MS2 Virus Like Particles (VLPs)** to be stable in the gut so they can be taken up by gut cells is essential for our Snaccine project. However, while MS2 VLPs have already been developed as vaccine technology (Su et al.), to the best of our knowledge no testing has previously been done for their **stability under the gut environment**. Thus, it is crucial to have a framework to their structural integrity and ensure they can stay stable under variable conditions, both in the cytosol as well as the dynamic gut conditions.

  

To address this gap, we developed this modelling module with the sole goal of examining how different **salt conditions** and **temperatures** affect the structural properties of the MS2 VLP. While VLP stability could be empirically tested in the lab, such screening experiments are often time consuming and do not provide mechanistic understanding. To fully capture the complexity of the MS2 molecular structure we opted to use atomistic **Molecular Dynamics (MD)**. Our MD platform provides a framework for better understanding the stability of MS2 VLPs and for the study of future modifications, such as Cell Penetrating Peptides (which would assist the uptake of VLPs by surrounding cells).

<Dropdown header="Procedure" level={0}>

---

## Procedure

### Overview of the procedure

We performed **MD simulation** on a hexamer section of an **MS2 VLP**.  
This section contains all relevant dimer types of the MS2 VLP.  

Using the **Amber** software package, we parameterized our system by introducing a representative protein force field, box dimensions, solvation, as well as initializing ion amounts corresponding to the 8 different salt concentrations we aimed to examine.  

Following this step, we ran a standard MD production pipeline with the **GROMACS** software package. At this step, we also set up the three different temperature conditions intended for testing.  

Using the same software, we then analyzed our MD output for its **RMSD**, **RMSF**, **Hydrogen bonds**, as well as the **interaction energy** between the constitutive dimers, to get insight into any structural changes observed over the conditions we tested.

### General outline of implementation

1. Set up an MS2 hexamer (2 AB dimers and 1 CC dimer) structure configuration  
2. Introduce necessary simulation parameters (e.g., box parameters, solvation, ion amount, force field, temperature)  
3. Perform energy minimization and equilibration  
4. Run the MD simulation  
5. Post-process resulting trajectories  
6. Analyze simulation (e.g., RMSD, RMSF, H-bonds, Energy)

<div style={{ borderLeft: '4px solid #ccc', paddingLeft: '16px', marginBottom: '24px' }}>
  <figure style={{ textAlign: 'center' }}>
    <img
      src="https://static.igem.wiki/teams/5649/figures/overview-dry-lab.webp"
      alt="Overview Dry Lab"
      width="600"
      style={{ display: 'inline-block', maxWidth: '100%', height: 'auto' }}
    />
    <figcaption><em>Figure 1: Overview of our MD procedure</em></figcaption>
  </figure>
</div>

---

## Description of the system

The MS2 VLPs are icosahedral capsids composed of 180 monomers in three different conformations A, B or C. These monomers form dimers, either as heterodimeric AB or as homodimer CC. The presence of both AB and CC is crucial for the structure to properly self-assemble. The different dimers form regular patterns with 5-fold symmetry interactions between AB dimers and 6-fold symmetry interactions between AB and CC dimers. While a complete picture of the structural stability of this structure would only be achieved by simulating a whole VLP such a task is computationally far too demanding to capture with MD. However, since an MS2 VLP is composed of only two types of dimers with two types of dimer-dimer interactions, an informative picture of this structure’s properties can be captured in a hexamer containing two AB and one CC dimer. A hexamer like this contains 774 residues which is computationally feasible for MD.
<div style={{ borderLeft: '4px solid #ccc', paddingLeft: '16px', marginBottom: '24px' }}>
  <figure style={{ textAlign: 'center' }}>
    <img
      src="https://static.igem.wiki/teams/5649/figures/death-star.webp"
      alt="Death Star"
      width="500"
      style={{ display: 'inline-block', maxWidth: '100%', height: 'auto' }}
    />
    <figcaption><em>Figure 2: PyMol visualization of MS2 VLP in green, with our hexamer section visualized in orange as cartoon</em></figcaption>
  </figure>
</div>

Next, we specified the condition types to be tested in this module. While the gut can contain a diverse and dynamic ion content, we recognized **sodium ions** as one of the most essential ions whose concentration can vastly differ in different gut regions and different physiological states, from as high as 150 mM to as low as 30 mM [2]. Thus, we decided to screen 8 concentrations in this range: **30 mM, 40 mM, 50 mM, 60 mM, 80 mM, 100 mM, 120 mM** and **150mM** of sodium. Additionally, temperature can potentially influence the stability of VLPs and thus we decided to explore 3 relevant temperature conditions for our concept: <Temperature celsius="18°C" fahrenheit="64°F" />, which is the temperature at which we successfully induced *E. coli* to produce VLPs; <Temperature celsius="37°C" fahrenheit="99°F" />, as the optimal *E. coli* growth condition as well as the body temperature of humans; and <Temperature celsius="42°C" fahrenheit="108°F" />, the body temperature of chickens.

---

## Parametrization

To set up the parametrization for an MD simulation of the MS2 hexamer, the **tleap** tool from AmberTools 24.8<Footnote noteKey="amber-tools" /> was used. To model the MS2 hexamer, the **ff19SB** force field <Footnote noteKey="ff19SB" /> was used. The structure was solvated in a box of water, modelled with the explicit **OPC water model**<Footnote noteKey="opc-water" />, padding the hexamer with 10 Å. Different neutralizing ion amounts were added, with the 8 already described salt concentrations or 150 mM if a temperature condition was being screened. The resulting parametrized Amber files were further converted to GROMACS format using **ParmEd 4.3.0**<Footnote noteKey="sampl5-engines"/>.

---

## MD production

Next, using the **GROMACS 2024.1** <Footnote noteKey="sampl5-engines"/> software suite, an MD production, post-processing and analysis were performed. All these processes were run on the Phase 2 **DelftBlue** supercomputer. As a first step of the production process, an energy minimization was carried out until a maximum force of 1000 kJ mol−1 nm−1 was reached using the **steepest descent algorithm**. In this process, equations of motion were numerically integrated with a time step of 2 fs using the **Verlet leapfrog algorithm**. Molecular coordinates were updated every 10 ps. Van der Waals and short-range electrostatic interactions were described with a cutoff of 1 nm. Additionally, to calculate long-range electrostatic interactions, we used the **particle-mesh Ewald summation** with fourth-order cubic interpolation and a grid spacing of 0.16 nm.

Furthermore, the system was equilibrated for 100 ps in the **NVT ensemble**. This was achieved using the **modified Berendsen thermostat** until 300 K equilibrium temperature was reached if testing for salt conditions otherwise the corresponding 3 screening temperatures were used (315 K, 310 K or 291 K). Following this, the hexamer system was equilibrated in the **NpT ensemble** to pressure of 1 bar by implementing a **Parrinello-Rahman barostat**  [  ]. At this stage the system went through production for approximately 600 ns. During this process hydrogen bonds were constrained using the **LINCS algorithm**  [  ].

---
## Postprocessing
To account for the periodic boundary conditions, standard for MD, the command `gmx trjconv -pbc` with the *whole* and *no_jump* options were used on the raw MD production output. Using `gmx trjconv -skip`, the processed MD were additionally compressed by a factor of 100 or 1000 for the resulting output to be used when dealing with more computationally demanding visualization or analysis.

---
## Analysis
As part of analyzing the resulting MD simulation the **Root Mean Square Deviation (RMSD)** and **Hydrogen Bonds** of the total protein as well as the **Root Mean Square Fluctuations (RMSF)** of the protein-backbone atoms were all measured over the whole uncompressed MD output. Three replicate simulations for each condition tested were analyzed. Additionally, to measure the total binding energies between dimers (both for AB-AB and AB-CC), the **Generalized Born and Surface Area continuum solvation** method was used through the **gmxMMPBSA** environment [  ].

---
</Dropdown>

<Dropdown header="Results" level={0}>


### Visualization and overview
 <div>
<iframe 
  src="https://video.igem.org/videos/embed/6v5aZJDh5x8jFNMBEmUnXH?loop=1&autoplay=1&muted=1"
  width="600"
  height="330"
  frameBorder="0"
  allowFullScreen
  style={{ display: 'block', margin: '0 auto' }}
></iframe>
</div>

### RMSD

As part of our analysis pipeline we first measured the total RMSD of our MS2 hexamer. This quantity allows us to track if a native stable state is reached of the system and validate whether the observed structural properties reflect the molecular system rather than being artifacts of equilibration. Additionally, RMSD trends can generally reveal large-scale conformational changes, domain motions, or relaxation processes occurring during the trajectory.

For all conditions a plateau was reached in the first 200 ns while already in the first 20 ns the system relaxed out from its initial equilibrated state. No noticeable differences were observed for the different conditions regarding RMSD. For all conditions tested there were no Fig signs of large scale conformational changes, as expected for this structure.

<figure style={{ textAlign: 'center' }}>
  <img
    src="https://static.igem.wiki/teams/5649/figures/ms2-rmsd.webp"
    alt="MS2 RMSD"
    width="600"
    style={{ display: 'inline-block', maxWidth: '100%', height: 'auto' }}
  />
  <figcaption><em>Figure 3: RMSD plot overlay for the different temperatures (top) and salt conditions (bottom) tested. Standard Deviation is visualized with semitransparant coloring.</em></figcaption>
</figure>

---

### Hydrogen bonds

We analyzed the total number of hydrogen bonds formed in the structure. We visualized this quantity through a histogram that shows the distribution of the number of hydrogen bonds from the whole timespan of the MD simulation. Not surprisingly all of them follow a Gaussian distribution. For most salt conditions no apparent changes were observed. However, interestingly for the lowest concentration of 30 mM a noticeably lower mean peak was observed. On the other hand, the temperature showed visible effect on the hydrogen bond distribution. While <Temperature celsius="37°C" fahrenheit="99°F" /> and <Temperature celsius="42°C" fahrenheit="108°F" /> showed relatively similar distributions, the <Temperature celsius="18°C" fahrenheit="64°F" /> noticeably shifted the peak of the Gaussian to the right. This reflects more persistent hydrogen bond formation as expected when thermal fluctuations are less present.

<figure style={{ textAlign: 'center' }}>
  <img
    src="https://static.igem.wiki/teams/5649/figures/hbonds-histo-salt-overlay.webp"
    alt="Hbonds Histo Salt Overlay"
    width="700"
    style={{ display: 'inline-block', maxWidth: '100%', height: 'auto' }}
  />
  <figcaption><em>Figure 4: Hydrogen bond Distribution Salt Overlay</em></figcaption>
</figure>


<Dropdown header="Expand to see separate salt conditions" level={0}>
<figure style={{ textAlign: 'center' }}>
  <img
    src="https://static.igem.wiki/teams/5649/figures/hbonds-histo-salt.webp"
    alt="Hbonds Histo Salt"
    width="700"
    style={{ display: 'inline-block', maxWidth: '100%', height: 'auto' }}
  />
  <figcaption><em>Figure 5: Separate Hbond Distribution histograms for each salt condition</em></figcaption>
</figure>
</Dropdown>

<figure style={{ textAlign: 'center' }}>
  <img
    src="https://static.igem.wiki/teams/5649/figures/hbonds-histo-temp.webp"
    alt="Hbonds Histo Temp"
    width="700"
    style={{ display: 'inline-block', maxWidth: '100%', height: 'auto' }}
  />
  <figcaption><em>Figure 6: Hbond distribution histograms for each temperature condition and overlay</em></figcaption>
</figure>

---

### Backbone RMSF

We analyzed the RMSF over all residues of the 6 MS2 protein backbone chains to study what is the influence of salt concentration and temperature on them. For all conditions, the observed RMSF aligned consistently with what is known about the general secondary structure of MS2. Protein regions with well-defined secondary structure such as alpha helices or beta strands aligned with regions of low fluctuation, while peaks of fluctuation aligned with terminal regions or loops between structured elements. These observations further validated our MD procedure.

In our RMSF analysis, we observe that one of the regions with higher fluctuation levels over all chains and conditions was the **AB loop**, colored in cyan on Fig. # and represented by the first RMSF peak after the N-terminal region. These loops are of particular relevance as they face the outer side of the viral capsid and can thus be a future target for **Cell Penetrating Peptide (CPP)** insertion. A CPP insertion like this can facilitate the uptake of VLPs by gut cells and induce immune response. Exactly because of their outer location they would be expected to have higher fluctuation levels.

<figure style={{ textAlign: 'center' }}>
  <img
    src="https://static.igem.wiki/teams/5649/figures/rmsf-salty2.webp"
    alt="RMSF Salty2"
    width="800"
    style={{ display: 'inline-block', maxWidth: '100%', height: 'auto' }}
  />
  <figcaption><em>Figure 7: RMSF plots for different salt conditions. a) Coloured PyMol structure of an MS2 hexamer as a reference. b) RMSF plot chain A1 and c) B2.</em></figcaption>
</figure> 

Interestingly when examined over the screened temperature conditions, the AB loops showed noticeably higher fluctuations for <Temperature celsius="37°C" fahrenheit="99°F" /> and <Temperature celsius="42°C" fahrenheit="108°F" /> compared to <Temperature celsius="18°C" fahrenheit="64°F" />. This is not particularly surprising as higher temperature would directly result in more thermal fluctuations and thus higher RMSF in more flexible regions of the protein. Surprisingly, however most often <Temperature celsius="42°C" fahrenheit="108°F" /> didn't result in noticeably higher fluctuations than <Temperature celsius="37°C" fahrenheit="99°F" />. In fact in most cases MS2 was observed to have a higher fluctuations for <Temperature celsius="37°C" fahrenheit="99°F" />. In contrast however, in the B2 monomer all temperatures showed indistinguishable levels of fluctuation (see Fig.). A similar pattern, was observed for other highly fluctuating regions in the MS2 backbone.

<figure style={{ textAlign: 'center' }}>
  <img
    src="https://static.igem.wiki/teams/5649/figures/rmsf-temp2.webp"
    alt="RMSF Temp2"
    width="700"
    style={{ display: 'inline-block', maxWidth: '100%', height: 'auto' }}
  />
  <figcaption><em>Figure 8: RMSF plots for different temperature conditions. a) RMSF plot chain B2 and b) C1.</em></figcaption>
</figure>

Somewhat similar to the temperature response, no straightforward relationship between RMSF and the salt concentration was observed. However, interestingly, 50 mM salt concentration seems to result in the highest level of fluctuation for most chains and B2 in particular. The fluctuations increase for the first three lowest salt concentrations and then begin to drop down after 50 mM, but around 80 mM again begin to steadily rise.

<figure style={{ textAlign: 'center' }}>
  <img
    src="https://static.igem.wiki/teams/5649/figures/rmsf-ab-ab.webp"
    alt="RMSF AB AB"
    width="750"
    style={{ display: 'inline-block', maxWidth: '100%', height: 'auto' }}
  />
  <figcaption><em>Figure 9: Plot of RMSF average over the AB loop region (11-17 residues) against salt concentration.</em></figcaption>
</figure>

Another interesting observation from the RMSF plot is that a wide peak of fluctuation is observed approximately between residue 66 and 80 (marked green on Fig. #). In A and C monomers, this region surprisingly contains regularly structured beta strands with a short three amino acid beta loop in between them. However, despite the regular secondary structure of this region, it is positioned on the side of the MS2 hexamer and would take part with other MS2 hexamers in the overall VLP structure. Thus this pinpoints a limitation of our hexamer simplification, specifically for this region of the MS2 backbone. Since there are no neighboring hexamers this beta strand structure can freely move and thus has high fluctuation levels. However, in a complete VLP assembly this would not be expected as this region would interact with the same protein structure of other neighboring MS2 dimers.

---

### Interaction energy measurement


To gain more direct insight into what is the influence of our tested conditions on the stability of this hexamer structure, we measured the interaction energy between individual dimers of the hexamer. The average of the measured energy was taken for 2 or 3 replicate trajectories. Thus, this quantity directly reflects how stable the interaction is between dimers. To measure this, we used the **gmxMMPBSA** environment. We measured the total free energy for both the AB–AB as well as the AB–CC interaction over all examined salt and temperature conditions.

  

  

<figure  style={{ textAlign:  'center' }}>

  

<img

  

src="https://static.igem.wiki/teams/5649/figures/overview-interaction.webp"

  

alt="Overview Interaction"

  

width="500"

  

style={{ display:  'inline-block', maxWidth:  '100%', height:  'auto' }}

  

/>

  

<figcaption><em>Figure 10: Overview of total interaction energies tested. 1. AB-AB interaction 2. AB-CC interaction.</em></figcaption>

  

</figure>

  

  

Over the measured temperatures and salt concentrations, the  AB-CC interaction energy was observed to be relatively stable with no evident trend lines. However, a trendline is observed for the AB-AB interaction energy with a slight increase with increasing salt concentration or temperature. This suggests that there could be a slight destabilizing effect when VLPs are exposed to chicken body temperature or higher. However, further per-residue energy measurement would need to be performed to gain more insight as to what kind of mechanism is resulting in this observed behaviour. 
  
<figure  style={{ textAlign:  'center' }}>

  

<img

  

src="https://static.igem.wiki/teams/5649/figures/mmpbsa-finalish.webp"

  

alt="MMPBSA"

  

width="750"

  

style={{ display:  'inline-block', maxWidth:  '100%', height:  'auto' }}

  

/>

  

<figcaption><em>Figure 11: AB-AB (top) and AB-CC (bottom) interaction Energy for different salt concentrations (left) and temperature (right).</em></figcaption>

  

</figure>

  
  




</Dropdown>

  
  

<Dropdown  header="Conclusion and Future Outlook"  level={0}>

### **Conclusion**

  

With our MD MS2 platform, we established a computational framework to evaluate how **temperature** and **salt concentration** influence the **structural properties of MS2 VLPs**.

  

Our simulations aligned with the already expected structural properties of the MS2 protein and additionally revealed that the MS2 hexamer remains globally stable across all tested conditions, based on its observed trajectory, bonds, and residual fluctuations. However, based on our interaction energy measurements between AB dimers, the simulated chicken gut conditions could have a potential destabilizing effect on the MS2 VLP structure, specifically from the AB-AB interaction. However, a more detailed energetic analysis — particularly at the residue level — is required to identify specific regions or interactions driving this instability. This residue-scale approach will constitute the next step in our analysis. In addition, no such relation was observed for the more energetically favourable AB-CC interactions, which could potentially provide enough support for the VLPs to remain stable even for suboptimal conditions for the AB-AB interaction. 

  

The robustness of MS2 VLPs can likely be enhanced through rational design strategies, and this would only be possible with a clear structural understanding that platforms like ours can provide. Future work could explore capsid mutations that improve inter-dimer interactions, enhance hydrophobic packing so and increase tolerance to salt variation. In particular, peptide insertions should also be further studied with our framework to study how they would further influence the stability of VLPs.


Thus, we successfully created a structural stability framework and were already able to gain insight into what could be the next steps in the development of our Snaccine concept.

  

---

  
  

### **Future Outlook**

  

Building on our platform, our next steps include:

  

1.  **Residue-level free energy decomposition** using MMPBSA and normal-mode analysis to pinpoint specific amino acids driving destabilization.

2.  **In silico mutagenesis and peptide-insertion screening**, testing how different protein m**Cell-Penetrating Peptides (CPPs)** or stabilizing linker residues influence structural integrity.

3.  **Full-capsid coarse-grained simulations**. While our model still provides important structural insight, our framework could be fed into a coarse-grained MD of the complete VLP, and study the structural properties of the complete VLP

4.  **Experimental validation** As we successfully managed to produce our own VLPs, a logical next step is to measure the degradation level of VLPs when exposed to different conditions and compare them to our MD results.


</Dropdown>
<FootnoteDefinition noteKey="ambertools">D. A. Case, H. M. Aktulga, K. Belfon, et al., “AmberTools,” Journal of Chemical Information and Modeling 63, no. 20 (2023): 6183–6191.</FootnoteDefinition>

<FootnoteDefinition noteKey="opc-water">S. Izadi, R. Anandakrishnan, and A. V. Onufriev, “Building Water Models: A Different Approach,” Journal of Physical Chemistry Letters 5, no. 21 (2014): 3863–3871.</FootnoteDefinition>

<FootnoteDefinition noteKey="lj-type-ion">P. Li, L. F. Song, and K. M. Merz, “Parameterization of Highly Charged Metal Ions Using the 12-6-4 LJ-Type Nonbonded Model in Explicit Water,” Journal of Physical Chemistry B 119, no. 3 (2015): 883–895.</FootnoteDefinition>

<FootnoteDefinition noteKey="ff19sb">C. Tian, K. Kasavajhala, K. A. A. Belfon, et al., “ff19SB: Amino-Acid-Specific Protein Backbone Parameters Trained Against Quantum Mechanics Energy Surfaces in Solution,” Journal of Chemical Theory and Computation 16, no. 1 (2020): 528–552.</FootnoteDefinition>

<FootnoteDefinition noteKey="bsc0-nucleic-acids">M. Zgarbová, M. Otyepka, J. Poner, et al., “Refinement of the Cornell et al. Nucleic Acids Force Field Based on Reference Quantum Chemical Calculations of Glycosidic Torsion Profiles,” Journal of Chemical Theory and Computation 7, no. 9 (2011): 2886–2902.</FootnoteDefinition>

<FootnoteDefinition noteKey="rna-protein-polarizability">H. Baltrukevich and P. Bartos, “RNA–Protein Complexes and Force Field Polarizability,” Frontiers in Chemistry 11 (2023): 1217506.</FootnoteDefinition>

<FootnoteDefinition noteKey="sampl5-engines">M. R. Shirts, C. Klein, J. M. Swails, et al., “Lessons Learned From Comparing Molecular Dynamics Engines on the SAMPL5 Dataset,” Journal of Computer-Aided Molecular Design 31, no. 1 (2017): 147–161.</FootnoteDefinition>

<FootnoteDefinition noteKey="gromacs-1995">H. Berendsen, D. Van Der Spoel, and R. Van Drunen, “GROMACS: A Message-Passing Parallel Molecular Dynamics Implementation,” Computer Physics Communications 91, no. 1–3 (1995): 43–56.</FootnoteDefinition>

<FootnoteDefinition noteKey="gromacs-2001">E. Lindahl, B. Hess, and D. Van Der Spoel, “GROMACS 3.0: A Package for Molecular Simulation and Trajectory Analysis,” Journal of Molecular Modeling 7, no. 8 (2001): 306–317.</FootnoteDefinition>

<FootnoteDefinition noteKey="gromacs-2005">D. Van Der Spoel, E. Lindahl, B. Hess, G. Groenhof, A. E. Mark, and H. J. C. Berendsen, “GROMACS: Fast, Flexible, and Free,” Journal of Computational Chemistry 26, no. 16 (2005): 1701–1718.</FootnoteDefinition>

<FootnoteDefinition noteKey="gromacs-2008">B. Hess, C. Kutzner, D. Van Der Spoel, and E. Lindahl, “GROMACS 4: Algorithms for Highly Efficient, Load-Balanced, and Scalable Molecular Simulation,” Journal of Chemical Theory and Computation 4, no. 3 (2008): 435–447.</FootnoteDefinition>

<FootnoteDefinition noteKey="gromacs-2013">S. Pronk, S. Páll, R. Schulz, et al., “GROMACS 4.5: A High-Throughput and Highly Parallel Open Source Molecular Simulation Toolkit,” Bioinformatics 29, no. 7 (2013): 845–854.</FootnoteDefinition>

<FootnoteDefinition noteKey="gromacs-2015">M. J. Abraham, T. Murtola, R. Schulz, et al., “GROMACS: High Performance Molecular Simulations Through Multi-Level Parallelism From Laptops to Supercomputers,” SoftwareX 1-2 (2015): 19–25.</FootnoteDefinition>

<FootnoteDefinition noteKey="exascale-challenges">S. Markidis and E. Laure, eds., *Solving Software Challenges for Exascale: International Conference on Exascale Applications and Software, EASC 2014, Stockholm, Sweden, April 2–3, 2014, Revised Selected Papers* (Lecture Notes in Computer Science), vol. 8759 (Springer International Publishing, 2015).</FootnoteDefinition>

<FootnoteDefinition noteKey="delftblue">Delft High Performance Centre (DHPC), *DelftBlue Supercomputer (Phase 2)* (2024).</FootnoteDefinition>

<FootnoteDefinition noteKey="pme-ewald">T. Darden, D. York, and L. Pedersen, “Particle Mesh Ewald: An N Log(N) Method for Ewald Sums in Large Systems,” Journal of Chemical Physics 98, no. 12 (1993): 10089–10092.</FootnoteDefinition>

<FootnoteDefinition noteKey="parrinello-rahman">M. Parrinello and A. Rahman, “Polymorphic Transitions in Single Crystals: A New Molecular Dynamics Method,” Journal of Applied Physics 52, no. 12 (1981): 7182–7190.</FootnoteDefinition>

<FootnoteDefinition noteKey="lincs">B. Hess, H. Bekker, H. J. C. Berendsen, and J. G. E. M. Fraaije, “LINCS: A Linear Constraint Solver for Molecular Simulations,” Journal of Computational Chemistry 18, no. 12 (1997): 1463–1472.</FootnoteDefinition>

<FootnotesList title="References"/>
</Content>

</FootnotesProvider>
